CONTRIBUTING.md

```diff
@@ -0,0 +1,244 @@
+
+More complex demonstrations of functionality are also welcome, but for now belong in community tutorials or template games.
+
+Check out [awesome-bevy](https://github.com/bevyengine/awesome-bevy) for a place to put your tutorials, tools, templates and crates!
+
+### Reviewing others work
+
+With the sheer volume of activity in Bevy's community, reviewing others work and helping it improve is one of the most valuable things you can do.
+You don't need to be an Elder Rustacean to be useful here: anyone can catch issues of missing tests, unclear docs, logic errors or so on.
+If you have unusual skills (e.g. advanced familiarity with `unsafe` code, rendering knowledge or web development experience) or personal experience with a problem, try to prioritize those areas to ensure we can get appropriate expertise where we need it.
+
+Focus on giving constructive, actionable feedback that results in real improvements to code quality or end-user experience.
+If you don't understand why an approach was taken, please ask!
+
+Small changes work well as comments or in-line suggestions on specific lines of codes.
+Larger changes deserve a comment in the main thread, or a pull request to the original author's branch (but mention that you've made one).
+When in doubt about a matter of architectural philosophy, refer back to **What we're trying to build** for guidance.
+
+Once you're happy with the work and feel you're reasonably qualified to assess quality in this particular area, leave your `Approved` review on the PR so we can mark it as `ready-for-cart`.
+
+There are a two main places you can check for new work to review:
+
+1. Pull requests on `bevy` and the `bevy-website` repos.
+2. [RFCs](https://github.com/bevyengine/rfcs), which need extensive thoughtful community input on their design.
+
+Official focus areas and work done by @cart go through this review process as well.
+Not even our project lead is exempt from reviews and RFCs!
+By giving feedback on this work (and related supporting work), you can help us make sure our releases are both high-quality and timely.
+
+Finally, if nothing brings you more satisfaction than seeing every last issue tagged and all resolved issues closed, feel free to message @cart for a Bevy org role to help us keep things tidy.
+As discussed in **How we're organized**, this is not intended to have a high bar.
+
+### Contributing your own ideas
+
+As discussed in **How we work together** Bevy is actively open to new ideas and serious contributions from outside community members.
+If you're new to Bevy, here's the workflow we use:
+
+1. Fork the `bevyengine/bevy` repository on GitHub, you'll need to create a GitHub account if you don't have one already.*
+2. Make your changes in a local clone of your fork, typically in its own new branch.
+3. For a higher chance of CI passing the first time, consider locally running `cargo run -p ci`. You can run the commands manually:
```

Comment:
Consider also adding info about markdownlint e.g. for changes in a readme. See https://github.com/bevyengine/bevy-website/pull/141

Reply:
Aha, I didn't think to check the open PRs for this info!